src/source.rs

```diff
@@ -56,6 +56,7 @@ pub(crate) mod sse;
 pub(crate) mod stdin;
 pub(crate) mod tcp;
 pub(crate) mod udp;
+pub mod unix_socket;
```

Comment:
I think we should use conditional compilation here to only include this one if built for a unix target.